cfme/tests/infrastructure/test_webmks_console.py

```diff
@@ -174,3 +174,132 @@ def _validate_login():
+    are being tested manually. Originally, there was one testcase for every combination, this
+    approach reduces number of needed testcases.
+
+    Testing matrix:
+        Systems:
+            - fedora latest
+            - rhel 6.x
+            - rhel 7.x
+            - windows 2012
+            - windows 7
+            - windows 10
+        Browsers:
+            - firefox latest
+            - chrome latest
+            - edge latest
+            - explorer 11
+        Providers:
+            - vsphere 6
+            - vsphere 65
+            - vsphere 67
+
+    Importance:
+        High:
+            - fedora latest + firefox, chrome + vsphere 67
+            - rhel 7.x + firefox, chrome + vsphere 67
+            - windows 10 + firefox, chrome, edge + vsphere 67
+        Medium:
+            - everything else
+
+    Testcases:
+        - test_webmks_console_firefox_vsphere67_win10
+        - test_webmks_console_firefox_vsphere67_rhel7x
+        - test_webmks_console_firefox_vsphere67_fedora
+        - test_webmks_console_firefox_vsphere67_win7
+        - test_webmks_console_firefox_vsphere67_win2012
+        - test_webmks_console_firefox_vsphere65_rhel7x
+        - test_webmks_console_firefox_vsphere65_fedora
+        - test_webmks_console_firefox_vsphere65_win2012
+        - test_webmks_console_firefox_vsphere65_win7
+        - test_webmks_console_firefox_vsphere65_win10
+        - test_webmks_console_firefox_vsphere6_win10
+        - test_webmks_console_firefox_vsphere6_win7
+        - test_webmks_console_firefox_vsphere6_win2012
+        - test_webmks_console_firefox_vsphere6_fedora
+        - test_webmks_console_firefox_vsphere6_rhel7x
+        - test_webmks_console_chrome_vsphere67_rhel7x
+        - test_webmks_console_chrome_vsphere67_fedora
+        - test_webmks_console_chrome_vsphere67_win10
+        - test_webmks_console_chrome_vsphere67_win7
+        - test_webmks_console_chrome_vsphere67_win2012
+        - test_webmks_console_chrome_vsphere65_rhel7x
+        - test_webmks_console_chrome_vsphere65_win7
+        - test_webmks_console_chrome_vsphere65_fedora
+        - test_webmks_console_chrome_vsphere65_win10
+        - test_webmks_console_chrome_vsphere65_win2012
+        - test_webmks_console_chrome_vsphere6_win2012
+        - test_webmks_console_chrome_vsphere6_win7
+        - test_webmks_console_chrome_vsphere6_rhel7x
+        - test_webmks_console_chrome_vsphere6_win10
+        - test_webmks_console_chrome_vsphere6_fedora
+        - test_webmks_console_edge_vsphere67_win10
+        - test_webmks_console_edge_vsphere65_win10
+        - test_webmks_console_edge_vsphere6_win10
+        - test_webmks_console_ie11_vsphere67_win2012
+        - test_webmks_console_ie11_vsphere67_win7
+        - test_webmks_console_ie11_vsphere65_win7
+        - test_webmks_console_ie11_vsphere65_win2012
+        - test_webmks_console_ie11_vsphere6_win2012
+        - test_webmks_console_ie11_vsphere6_win7
```

Comment:
@apagac overall, I like the approach but I have one concern. 
There are a lot of cases in one case, will it be easy to track which ones are passed already during execution?

Reply:
I know this has been merged, still want to get back to this question.

So all console tests are run manually due to huge number of combinations of browsers and systems. If some of the tests fail, I can always make a note in polarion when failing this whole testcase and put there that particular combination that failed.

In the future we might consider using `pytest_generate_tests` with a function that would take browser, provider and operating system and would generate the tests for us.

Reply:
@apagac thanks! Sounds good :)